### PR TITLE
[API-1046] Document delete endpoint

### PR DESCRIPTION
**Pull request by nathansandi**

### CODE REVIEW

File: versions/v2/content/pentests.md [hunk 1]
```diff
@@ -353,3 +353,30 @@ within Cobalt's API of the new pentest.
...
 Remember - you can only duplicate a pentest within the organization specified in the <code>X-Org-Token</code> header.
 </aside>
+
+## Delete a Pentest
+
+```sh
+curl -X DELETE 'https://api.cobalt.io/pentests/YOUR-PENTEST-IDENTIFIER' \
+  -H 'Accept: application/vnd.cobalt.v2+json' \
+  -H 'Authorization: Bearer YOUR-PERSONAL-API-TOKEN' \
+  -H 'Content-Type: application/vnd.cobalt.v2+json' \
```
Comment (msdundar) [inline versions/v2/content/pentests.md:363]: Do we need this for DELETE? The other DELETE example (_delete an asset_) doesn't have that.

File: versions/v2/content/pentests.md [hunk 2]
```diff
@@ -353,3 +353,30 @@ within Cobalt's API of the new pentest.
...
+```
+
+> The above command returns no data and a `204` response code when successful.
+
+This endpoint deletes a pentest belonging to the organization specified in the header.
+Note that you **can** only delete a pentest when it is in draft or review state.
+
+### HTTP Request
+
+`DELETE https://api.cobalt.io/pentests/YOUR-ASSET-IDENTIFIER`
```
Comment (msdundar) [inline versions/v2/content/pentests.md:374]:
```suggestion
`DELETE https://api.cobalt.io/pentests/YOUR-PENTEST-IDENTIFIER`
```

File: versions/v2/content/pentests.md (hunk 1 above)
Comment (inikolov89) [reply versions/v2/content/pentests.md:363]: Agree, `Content-Type` is for requests accepting body.

Comment (nathansandi) [reply versions/v2/content/pentests.md]: We have reason, I just retested to make sure, we dont need

**Review by msdundar — APPROVED** [review]

**Pull request merged**